tests/linter_test.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+from demisto_sdk.dev_tools.linter import Linter
+
+
+class TestLinter:
+    path = "./CommonServerPython.py"
+
+    def setup_class(self):
+        if os.path.isfile(self.path):
```

Comment:
if the test creates the file, this should be in teardown not setup.
if the file should be there regardless to the test, it shouldn't remove it (rename and revert at teardown)